package.json

```diff
@@ -77,5 +77,10 @@
   "repository": {
     "type": "git",
     "url": "git+https://github.com/wmde/query-builder.git"
+  },
+  "jest": {
+    "setupFiles": [
+      "./tests/unit/setEnvVars.js"
```

Comment:
Should that not be `.jest/setEnvVars.js`? I'm confused that the tests are still passing :thinking:

Reply:
Actually this was from another way I tried that didn't work. So this is actually not being used. Thanks for pointing it out.